tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/Script_Metadata.csv

```diff
@@ -150,7 +150,7 @@ WR,Name,Script_Code,Age,Size,Sample,Sample_Code,Origin Country,~Density,Likely L
 148,Old_Sogdian,Sogo,11.0,0,𐼙,10F19,Uzbekistan,1,Sogdian,sog,Exclusion,Yes,no,no,no,no
 149,Sogdian,Sogd,11.0,0,𐽂,10F42,Uzbekistan,1,Sogdian,sog,Exclusion,Yes,no,Yes,no,no
 150,Elymaic,Elym,12.0,0,𐿱,10FF1,Iran,1,Imperial Aramaic,arc,Exclusion,Yes,no,no,no,no
-151,Nyiakeng Puachue Hmong,Hmnp,12.0,0,𞄈,1E108,USA,1,Hmong Daw,mww,Limited_Use,no,no,no,no,no
+151,Nyiakeng Puachue Hmong,Hmnp,12.0,0,𞄈,1E108,USA,1,Green Hmong,hnj,Limited_Use,no,no,no,no,no

```

Comment:
@markusicu where does this data come from?  I'm not familalir with the script metadata process, but the data as-is seems to reject the `hnj_Hmnp_US` locale and instead expected `hnj_Hmnp_LA` and `mww_Hmng_US` to be the default contents for their respective script and langauge codes. Any ideas?

Reply:
I had at first tried to 'fix' this by adding to an exclusion list in the tests, but realized this was coming from the data.

Reply:
The script metadata comes from a spreadsheet that Mark started and that I have been maintaining for a while. I usually get changes reviewed by Debbie/Roozbeh/Ken/Anshuman/Mark. The spreadsheet is linked from http://cldr.unicode.org/development/updating-codes/updating-script-metadata

On the LikelyLanguage Sheet I see

Code | Script | Code | Language | Approx Lit Modern Pop (1st &amp; 2nd lang) | Comments/References
-- | -- | -- | -- | -- | --
Hmng | Pahawh_Hmong | hnj | Hmong Njua | &nbsp; | verify which Hmong lang code is correct; either mww (Hmong Daw) or hnj (Hmong Njua) per Wikipedia
Hmnp | Nyiakeng Puachue Hmong | mww | Hmong Daw | &nbsp; | &nbsp;

Unicode 13 is not very helpful for this question: https://www.unicode.org/versions/Unicode13.0.0/ch16.pdf
> The Pahawh Hmong script was originally devised by Shong Lue Yang in 1959 to write the Hmong language. The script was devised in Laos, and taken to refugee camps in northern Thailand. In the late 20th century, it then moved with waves of immigrants to Australia and the United States, where it remains in current use.

vs.
> Nyiakeng Puachue Hmong is a writing system created in the 1980s by Reverend Chervang Kong to write the White Hmong and Green Hmong languages. It is also called the Ntawv Txawjvaag or Chervang script, and was devised for use in the United Christians Liberty Evangelical church in the United States. It is reportedly used today in Laos, Thailand, Vietnam, France and Australia.

https://en.wikipedia.org/wiki/Pahawh_Hmong
> ... is an indigenous semi-syllabic script, invented in 1959 by Shong Lue Yang, to write two Hmong languages, Hmong Daw (Hmoob Dawb White Miao) and Hmong Njua AKA Hmong Leng (Moob Leeg Green Miao).

https://en.wikipedia.org/wiki/Nyiakeng_Puachue_Hmong
> ... is an alphabet script devised for White Hmong and Green Hmong ...

**but**

https://en.wikipedia.org/wiki/Hmong_language#Varieties_in_Laos seems to indicate that White Hmong (Hmong Daw) = mww is more common in *both* Laos and the US.

Reply:
Oh, and *do not* change the .csv file manually. Such changes would just be overwritten the next time someone (usually me) goes through the documented process.

Reply:
the `Hmnp` community requested to add `hnj_Hmnp_US`  (Green Hmong).   I sent out a query asking about White Hmong (mww).   They had previously requested that `US` be the default content.

Reply:
It should be possible to add data for any reasonable locale regardless of whether the script metadata says that the language is the default for the script -- without tests failing.

Reply:
this is back burner for a while, but i'll start with working around the test failure.